iis-workflows/iis-workflows-collapsers/src/main/resources/eu/dnetlib/iis/workflows/collapsers/basic_collapser/oozie_app/workflow.xml

```diff
@@ -14,49 +14,59 @@
 			<name>schema</name>
 			<description>input and output schema</description>
 		</property>
-        <property>
+		<property>
 			<name>output</name>
 			<description>output port</description>
 		</property>
-        <property>
+		<property>
 			<name>record_collapser</name>
-            <value>eu.dnetlib.iis.workflows.collapsers.basic.BestFilledCollapser</value>
+			<value>eu.dnetlib.iis.workflows.collapsers.basic.BestFilledCollapser</value>
 			<description>record collapser class</description>
 		</property>
-        <property>
+		<property>
 			<name>significant_fields</name>
-            <value>$ALL$</value>
+			<value>$ALL$</value>
 			<description>comma-separated list of significant field names used in within origin collapser;
-                         @see eu.dnetlib.iis.workflows.collapsers.SignificantFieldsCollapser</description>
+				@see eu.dnetlib.iis.workflows.collapsers.SignificantFieldsCollapser
+			</description>
+		</property>
+		<property>
+			<name>mapred_child_java_opts</name>
+			<value>-Xmx2048M</value>
+			<description>java-opts, e.g. maximum heap size for oozie</description>
```

Comment:
Instead of "maximum heap size for oozie" it should be rather something like "maximum heap size for a single JVM running MapReduce".